examples_gbs/run_tutorial_sample.py

```diff
@@ -0,0 +1,140 @@
+# pylint: disable=invalid-name,no-member,wrong-import-position,wrong-import-order,ungrouped-imports
+"""
+Sampling Tutorial
+=================
+
+This tutorial provides a quick guide to generating samples from GBS. This functionality is
+provided in the :mod:`~.gbs.sample` module of the applications layer.
+
+Sampling from GBS
+-----------------
+
+A GBS device can be programmed to sample from any symmetric matrix :math:`A`. To sample,
+we must specify both the mean number of photons being generated in the device and the form of
+detection used at the output: threshold detection or photon-number resolving (PNR) detection.
+Threshold detectors are restricted to measuring whether or not one or more photons have arrived
+at the detector, whereas PNR detectors are able to count the number of photons.
+
+Let's take a look at both types of sample. We can generate samples from a random 5-dimensional
+matrix:
+"""
+
+from strawberryfields.gbs import sample
+import numpy as np
+
+modes = 5
+n_mean = 4
+samples = 5
+
+A = np.random.random((modes, modes))
+A = A + A.T
+
+s_thresh = sample.sample(A, n_mean, samples, threshold=True)
+s_pnr = sample.sample(A, n_mean, samples, threshold=False)
+
+print(s_thresh)
+print(s_pnr)
+
+##############################################################################
+# In each case, a sample is a sequence of integers of ``len(modes) = 5``. Threshold samples are

```

Comment:
I'm not sure how to read "of `len(modes) = 5" in English. Maybe " In each case, a sample is a sequence of integers of length five, i.e., ``len(modes) = 5`` "?